lib/ansible/executor/task_executor.py

```diff
@@ -1165,10 +1165,13 @@ def start_connection(play_context, variables, task_uuid):
         'ANSIBLE_NETCONF_PLUGINS': netconf_loader.print_paths(),
         'ANSIBLE_TERMINAL_PLUGINS': terminal_loader.print_paths(),
     })
+    verbosity = ''
+    if display.verbosity:
+        verbosity = '-%s' % ('v' * display.verbosity)
     python = sys.executable
     master, slave = pty.openpty()
     p = subprocess.Popen(
-        [python, ansible_connection, to_text(os.getppid()), to_text(task_uuid)],
+        [python, ansible_connection, verbosity, to_text(os.getppid()), to_text(task_uuid)],
```

Comment:
If `display.verbosity` is falsy, this results in `python anisble_connection '' <pid> <uuid>` and the extra parameter isn't understood to be intended as a flag. I think this (and the above change to verbosity) should fix the issue?
```suggestion
        [python, ansible_connection, *verbosity, to_text(os.getppid()), to_text(task_uuid)],
```